js/wp-seo-replacevar-plugin-302.js

```diff
@@ -8,9 +8,20 @@
 	var YoastReplaceVarPlugin = function() {
 		this.replaceVars = wpseoReplaceVarsL10n.replace_vars;
 
+		this.taxonomyRepository = {};
+		this.currentTaxonomies  = {};
+
 		YoastSEO.app.registerPlugin( 'replaceVariablePlugin', { status: 'ready' } );
 
 		this.registerModifications();
+
+		// Adding events to the dom, to fetch changes.
+		if ( jQuery( '#post_ID').val() !== undefined ) {
+			// Set events for each taxonomy box.
+			jQuery.each( jQuery( '.categorydiv' ),  this.setTaxonomyEvents.bind( this ) );
```

Comment:
jQuery has flavour for this:
`jQuery( '.categorydiv' ).each( this.setTaxonomyEvents.bind( this ) );`